catalog/cassandra/3.11/init-task-dockerfile/Dockerfile

```diff
@@ -29,6 +29,7 @@ RUN set -x \
 	&& export GNUPGHOME="$(mktemp -d)" \
 	&& gpg --keyserver ha.pool.sks-keyservers.net --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4 \
 	&& gpg --batch --verify /usr/local/bin/gosu.asc /usr/local/bin/gosu \
+	&& gpgconf --kill gpg-agent \
```

Comment:
Not sure why do we need this? GOSU installation is the same for all catalog services. This dockfile is almost the same with cassandra/3.11/dockerfile/Dockerfile. If you could build the cassandra docker image successfully, why the build fails for the init docker image?